common/src/main/java/whocraft/tardis_refined/common/tardis/TardisArchitectureHandler.java

```diff
@@ -37,7 +37,7 @@ public static void generateDesktop(ServerLevel operator, DesktopTheme theme) {
         for (Iterator<BlockPos> iterator = BlockPos.betweenClosed(corner, farCorner).iterator(); iterator.hasNext();) {
             BlockPos pos = iterator.next();
 
-            operator.setBlock(pos, Blocks.STONE.defaultBlockState(),1);
+            operator.setBlock(pos, BlockRegistry.GROWTH_STONE.get().defaultBlockState(),0);
```

Comment:
Why 0?

Reply:
We can use constants for setblock flags. There is a list in the vanilla ``Block`` class